_pages/rfc-unify-coc-enforcement.md

```diff
@@ -0,0 +1,70 @@
+# RFC - Unify CoC and Enforcement policies
+
+## tl;dr
+
+* Move the enforcement steps to the top of the CoC.
+* Keep the separate enforcement document as a detailed policy that includes lessons/values/nuance.
+* Stop directing people to both the CoC and enforcement policy: on the signup page, on docs, etc.
+* Put a link to the full enforcement values doc in the CoC.
+
+## Intro
+
+The first thing people should see when they go to the CoC is how it affects them. The bulk of the stuff on the CoC itself amounts to legalese that most users won't read -- so in the interest of conveying information that matters right away, it's best to lead with how CoC violations are handled. I would argue that's the main thing people even are about.
+
+Hopefully, these changes will make expectations even clearer for everone!
+
+## Changes
+
+### Move enforcement steps to CoC
+
+There's two sections of enforcement steps in the enforcement doc. Combine them, reword them, and put them on the very top of the main CoC.
+
+```
+If you see a CoC violation, these are the steps that should be followed:
+
+1. Tell the person “hey, that’s not ok”.
+2. That person should apologize immediately and correct the behavior.
+3. If this doesn’t happen, or if you're uncomfortable speaking up, [contact admins](#contacting-admins).
+4. As soon as available, an admin will join, identify themselves, and take [further action (see below)](#further-enforcement), starting with a warning, then temporary deactivation, then long-term deactivation.
+
+**The Admin team will prioritize the well-being and comfort of the recipients of the violation over the comfort of the violator.**
+```
+
+The `Enforcement` section in the CoC should be replaced with the following:
+
+```
+## Admin Enforcement
+
+Once the admins get involved, they will follow a documented series of steps and do their best to preserve the well-being of WeAllJS members. This section covers actual concrete steps. For further information/details on values and practices that admins will generally apply when enforcing the Code of Conduct, refer to [the full enforcement document](http://wealljs.org/enforcement).
+
+### Contacting Admins
+
+You may get in touch with the WeAllJS admin team through any of the following methods:
+
+  * Use the `/admin` command in the WeAllJS Slack to send a message directly to the admin channel.
+  * Direct Message [@WeAllJS](https://twitter.com/wealljs) on Twitter with the details and any relevant links.
```

Comment:
Can we order these by which we'd prefer?

I'd like:

- /admin
- DM in Slack
- Email
- DM WeAllJs